internal/controllers/topology/cluster/patches/variables/variables.go

```diff
@@ -118,6 +135,11 @@ func Global(clusterTopology *clusterv1.Topology, cluster *clusterv1.Cluster) (Va
 		variables[variable.Name] = variable.Value
 	}
 
+	clusterNetworkIPFamily, err := cluster.GetIPFamily()
+	if err != nil {
+		return variables, err
```

Comment:
```suggestion
		return nit, err
```
Let's please return nil instead of a partial result

Reply:
EDIT: `return nit, err` => `return nil, err` :joy: